site-src/index.md

```diff
@@ -96,16 +96,17 @@ standard API.
 
 ## What's the difference between Gateway API and an API Gateway?
 
-A Gateway is a networking component that allows data to transfer from one
+The concept of a Gateway is a networking component that allows data to transfer from one
 network to another network.
 
-An API Gateway is a Gateway that deals more specificially with backend services
+An API Gateway is a Gateway implementation that deals more specifically with backend services
 and routes client reqeusts to those backend services. Those services expose an
 API such as HTTP, REST or some other protocol on top of TCP or UDP.
 
 Gateway API is an interface, or set of resources, that model service networking
-in Kubernetes. One of the main resources is a `Gateway`. If you implement
-Gateway API, you are a Gateway provider. A `Gateway` resource declares the
+in Kubernetes. One of the main resources is a `Gateway`. As a Gateway Provider,
+you can implement the Gateway API to model Kubernetes service networking in an
+expressive, extensible, and role-oriented way. A `Gateway` resource declares the
```

Comment:
Can we combine these two sentences into one? 
"One of the main resources is a `Gateway`"
"A `Gateway` resource declares the Gateway type (or class) to instantiate and its configuration."

For example: "One of the main resources is a `Gateway` which declares the Gateway type (or class) to instantiate and its configuration."